README.md

```diff
@@ -12,6 +12,7 @@ Inspired by the [awesome](https://github.com/sindresorhus/awesome) list thing.
 * [Empty - The Codeless Code](http://thecodelesscode.com/case/6) - Qi
 * [How to Write a Git Commit Message](https://chris.beams.io/posts/git-commit/) - Chris Beams
 * [Steps to better code](https://medium.com/@isaaclyman/steps-to-better-code-e6c3cce0c7f9) - Isaac Lyman
+* [The 10 commandments of good source control management](https://www.troyhunt.com/10-commandments-of-good-source-control) - Troy Hunt
```

Comment:
Aiming to keep this list "lean", I don't think we need another source control article. And Troy article have tips like "stop using VSS", "Do not commit compilation output" and "Do not delegate your commits" - which is already ingrained in our culture, so it does not add much